client/modules/datafiles/src/AddFileFolder/AddFileFolder.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Button } from 'antd';
+import DatafilesModal from '../DatafilesModal/DatafilesModal';
+import { useAuthenticatedUser, useFileListingRouteParams } from '@client/hooks';
+import styles from './AddFileFolder.module.css';
+
+export const AddFileFolder: React.FC = () => {
+  const { api, system, path } = useFileListingRouteParams();
+  const { user } = useAuthenticatedUser();
+  return (
+    <ul
+      style={{
```

Comment:
Can you please do another pass on this diff to replace the remaining inline styles with css classes?